### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by shirady**
### Explain the changes
1. Update README.md

### Issues: Fixed #xxx / Gap #xxx
1. none

### Testing Instructions:
1. none

- [X] Doc added/updated
- [ ] Tests added

### CODE REVIEW

**Review by liranmauda — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -46,108 +65,171 @@ noobaa install
...
+Once a custom resource is installed, users can create, update and access its objects using [`kubectl`](https://kubernetes.io/docs/reference/kubectl/), just as they do for built-in resources.
+  - [NooBaaSystem](doc/noobaa-crd.md) - The basic CRD to deploy a NooBaa system. Represents a single installation of NooBaa that includes a set of sub-resources (backing-stores, bucket-classes, and buckets).
+  - [BackingStore](doc/backing-store-crd.md) - Storage resources. These storage targets are used to store deduplicated, compressed and encrypted chunks of data.
+  - [NamespaceStore](doc/namespace-store-crd.md) - Data resources. These storage targets are used to store and read plain data.
+  - [BucketClass](doc/bucket-class-crd.md) - Policies applied to a class of buckets, defines bucket policies relating to data placement.
+  - [Account](doc/noobaa-account-crd.md) - We use the account to receive new credentials set for accessing different noobaa services.
+- Bucket Claim:
+  - [OBC Provisioner](doc/obc-provisioner.md) - Although it using a library that is not active, we still have this option in our system.
+  - [COSI Provisioner](doc/obc-cosi-provisioner.md) - Using an active library.
```
Comment (guymguym) [inline README.md:211]: fix link. also fix description - COSI is a new kubernetes storage standard (like CSI) to provision object storage buckets.

File: README.md [hunk 2]
```diff
@@ -46,108 +65,171 @@ noobaa install
...
+Once a custom resource is installed, users can create, update and access its objects using [`kubectl`](https://kubernetes.io/docs/reference/kubectl/), just as they do for built-in resources.
+  - [NooBaaSystem](doc/noobaa-crd.md) - The basic CRD to deploy a NooBaa system. Represents a single installation of NooBaa that includes a set of sub-resources (backing-stores, bucket-classes, and buckets).
+  - [BackingStore](doc/backing-store-crd.md) - Storage resources. These storage targets are used to store deduplicated, compressed and encrypted chunks of data.
+  - [NamespaceStore](doc/namespace-store-crd.md) - Data resources. These storage targets are used to store and read plain data.
+  - [BucketClass](doc/bucket-class-crd.md) - Policies applied to a class of buckets, defines bucket policies relating to data placement.
+  - [Account](doc/noobaa-account-crd.md) - We use the account to receive new credentials set for accessing different noobaa services.
+- Bucket Claim:
+  - [OBC Provisioner](doc/obc-provisioner.md) - Although it using a library that is not active, we still have this option in our system.
```
Comment (guymguym) [inline README.md:210]: just say ObjectBucketClaim is currently the main CR to provision buckets, however it is being deprecated in favor of COSI.

**Review by guymguym — APPROVED** [review]
two minor comments

File: README.md (hunk 1 above)
Comment (shirady) [reply README.md]: Changed, thanks.

File: README.md (hunk 2 above)
Comment (shirady) [reply README.md]: Changed, thanks.

**Pull request merged**